junifer/preprocess/confounds/fmriprep_confound_remover.py

```diff
@@ -404,6 +423,14 @@ def _pick_confounds(self, input: dict[str, Any]) -> pd.DataFrame:
             out_df["spike"] = fd
             to_select.append("spike")
 
+        # add binary std_dvars regressor if needed at given threshold
```

Comment:
std_dvars should not be added as a regressor to perform scrubbing, but used to generate a mask that is then passed on to `signal.clean`

Reply:
So the mask would be temporal which should remove the timepoints from both BOLD image and confounds table and then pass the result to `nilearn.image.clean_img()`?